src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.html

```diff
@@ -22,18 +22,34 @@
     </span>
   </div>
   <div class="card-body fill-plain">
-    <div>
+    <form #courseForm="ngForm" (ngSubmit)="onSaveCourse()">
       <div class="form-row form-group">
         <label class="col-sm-3">Course ID:</label>
         <div class="col-sm-3">
-          <input class="col-xs-12 form-control" maxlength="40" tabindex="1" [(ngModel)]="course.courseId" disabled>
+          <input class="col-xs-12 form-control" type="text" tabindex="1"
+                 #courseId="ngModel" [(ngModel)]="course.courseId" name="courseId"
+                 [ngClass]="{'invalid' : courseId.invalid && (courseId.touched || courseId.dirty),
+                 'valid' : courseId.valid && (courseId.touched || courseId.dirty)}"
+                 [maxlength]="40" required disabled/>
```

Comment:
The slash `/` may not be necessary here? Maybe we should keep it consistent. Same in line 48.